app/Http/Controllers/PrintController.php

```diff
@@ -112,6 +116,18 @@ public function modifyBalance(Request $request) {
         $print_account->update(['last_modified_by' => Auth::user()->id]);
         $print_account->increment('balance', $balance);
 
+        $admin_checkout = Checkout::where('name', 'ADMIN')->firstOrFail();
+        Transaction::create([
```

Comment:
You should use Admin\AdminCheckoutController::createTransaction here. In order to do that, change the function to public static.